### ISSUE TITLE
Does linuxdeployqt also run on a Raspberry?

### ISSUE THREAD

**Issue by lm7805** [probonopd/linuxdeployqt#104]
Is it possible to somehow use linuxdeployqt on a Raspberry Pi 2 with Rapsbian? 
I got the error "cannot execute binary file: Exec format error"

I wrote my Qt Application with QtCreator on Raspbian and also want to run it on this system.

**Comment by probonopd** [comment]
You need to compile it for the `armv7l` architecture. I don't see why it shouldn't be possible, but I have not done it myself.

**Comment by probonopd** [comment]
@mweaverh has been working on running AppImageKit on such devices, perhaps he can share some insight that may be useful here, too.

**Comment by mweaverh** [comment]
I've had success bundling appimages on small devices. First had to build (and bundle) appimagetool on the device.  PR #104 will be useful. Happy to share details next week if you're interested.

**Comment by probonopd** [comment]
@mweaverh yes, we're definitely interested, possibly we could even get a cross compile build going on Travis CI?

**Comment by mweaverh** [comment]
It should be doable. Mine was not cross-compiled though.

What would also cool is an appimagetool cross-bundler. I've made no efforts in that direction yet, but may do so eventually unless someone beats me to it.

**Comment by lm7805** [comment]
@mweaverh Yes, details would be highly appreciated!

**Comment by mweaverh** [comment]
I haven't worked with linuxdeployqt, but I'm assuming it's similar.  

The key step is building `appimagetool` on the target machine.  Once I figured out how to compile appimagetool, and to bundle it (essential but not quite obvious), then all went smoothly.
After running `bash -ex build.sh` per instructions,
```
PATH=./out  out/appimagetool.AppDir/usr/bin/appimagetool  out/appimagetool.AppDir
```
The key is setting the path so that `appimagetool` picks up the local build of `mksquashfs`.

Not sure if this addresses your issue, LMK.
